R/checks.R

```diff
@@ -644,3 +625,11 @@ check_eval_time <- function(eval_time, metrics) {
   invisible(NULL)
 
 }
+
+check_time_limit_arg <- function(x, call = rlang::caller_env()) {
+  if (!inherits(x, c("logical", "numeric")) || length(x) > 1) {
```

Comment:
```suggestion
  if (!inherits(x, c("logical", "numeric")) || length(x) != 1L) {
```

As this ought not to be length-0 either, right?